inference-engine/src/inference_engine/cnn_network_ngraph_impl.cpp

```diff
@@ -337,10 +338,12 @@ CNNNetworkNGraphImpl::reshape(const std::map<std::string, std::vector<size_t>>&
         auto specialized_ngraph_function = cloneFunction(true);
```

Comment:
Please change this line to: 
```suggestion
        auto specialized_ngraph_function = cloneFunction(false);
```
We shouldn't call constant folding for 2 times.